controllers/good_morning_controller.py

```diff
@@ -1,26 +1,38 @@
-from discord import Interaction, Thread
+from discord import Client, Interaction, Thread
 from db import DB
 from config import YAMLConfig as Config
-from datetime import datetime
+from datetime import datetime, timedelta
 from time import mktime as epochtime
 from pytz import timezone
 import asyncio
+from controllers.temprole_controller import TempRoleController
+import logging
+from discord.ext import tasks
 
+LOG = logging.getLogger(__name__)
 
 STREAM_CHAT_ID = Config.CONFIG["Discord"]["Channels"]["Stream"]
 REWARD_ROLE_ID = Config.CONFIG["Discord"]["GoodMorning"]["RewardRole"]
 REWARD_REDEMPTION_CHANNEL_ID = Config.CONFIG["Discord"]["GoodMorning"][
     "RedemptionChannel"
 ]
+REWARD_PROGRESS_CHANNEL = Config.CONFIG["Discord"]["GoodMorning"]["ProgressChannel"]
```

Comment:
For future commits please include this new config value in `config.example.yaml`. I have gone ahead and committed this to save time on review.

Reply:
Will do!
Also gonna make a PR to add the missing config options to the example file, as I believe I had to add a few from errors.